lib/publiccloud/instance.pm

```diff
@@ -227,6 +227,7 @@ sub wait_for_ssh
     $args{timeout} //= 600;
     $args{proceed_on_failure} //= 0;
     $args{username} //= $self->username();
+    $args{public_ip} //= $self->public_ip();
```

Comment:
Why is this needed for enabling the boottime test?

Reply:
It's not.

Reply:
It's not.

Reply:
I see, it's needed for the ssh command above